packages/search-token-server/package.json

```
@@ -3,6 +3,10 @@
   "version": "1.13.0",
   "main": "server.ts",
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:coveo/cli.git"
+  },
```

Be specific on the `repository` meta-data, it'll help npm user to figure what's what more easily.
See https://github.com/coveo/ui-kit/pull/967 for example.
See https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository for doc ref.

```suggestion
  "repository": {
    "type": "git",
    "url": "git@github.com:coveo/cli.git",
    "directory": "packages/search-token-server"
  },
```